clang/test/CIR/CodeGen/pointer-to-data-member.cpp

```diff
@@ -12,13 +12,13 @@ struct Incomplete;
 // CHECK-DAG: !ty_22Incomplete22 = !cir.struct<struct "Incomplete" incomplete>
 
 int Point::*pt_member = &Point::x;
-// CHECK: cir.global external @pt_member = #cir.data_member<0> : !cir.data_member<!s32i in !ty_22Point22>
+// CHECK: cir.global external @pt_member = #cir.data_member<0, offset = 0> : !cir.data_member<!s32i in !ty_22Point22>
```

Comment:
Not sure I see the need for for `offset`. If we already have the member offset, you can just look at the original struct and call for `getElementOffset()`

Reply:
Cool. I didn't realize that we can query the data layout at lowering time. Removed the `#cir.data_member_ptr_layout` attribute.